examples/tensorflow-mnist-pixel-threshold/task-plugins/securingai_custom/evaluation/pixelthreshold.py

```diff
@@ -14,31 +14,28 @@
 #
 # ACCESS THE FULL CC BY 4.0 LICENSE HERE:
 # https://creativecommons.org/licenses/by/4.0/legalcode
-"""A task plugin module for the Pixel Threshold evasion attack.
 
+"""A task plugin module for the Pixel Threshold evasion attack.
 The Pixel Threshold [kotyan2019]_ is an evasion attack that attempts to
 fool a trained classifier by perturbing a test image within a certain threshold
 of pixels. This task plugin uses the Adversarial Robustness Toolbox's
 [art2019]_ implementation of the Pixel Threshold attack |pt_art|.
-
 References:
     .. [art2019] M.-I. Nicolae et al., "Adversarial Robustness Toolbox v1.0.0,"
         2019. [Online]. Available:
        `arXiv:1807.01069v4 [cs.LG] <http://arxiv.org/abs/1807.01069v4>`_.
-
     .. [kotyan2019] Kotyan, S., & Vasconcellos Vargas, D. (2019).
        Adversarial Robustness Assessment: Why both $ L_0 $ and $ L_\infty $ Attacks Are Necessary.
        2019. [Online].
        Available: `arXiv:1906.06026 [stat.ML] <https://arxiv.org/abs/1906.06026>`_.
-
     .. [kotyan2019] Su, J., Vargas, D. V., & Sakurai, K. (2019).
        One pixel attack for fooling deep neural networks.
        "IEEE Transactions on Evolutionary Computation", 2019. [Online].
        Available: `arXiv:1710.08864 [stat.ML] <https://arxiv.org/abs/1710.08864>`_.
```

Comment:
```suggestion
    .. [kotyan2019] Su, J., Vargas, D. V., & Sakurai, K. (2019).
       One pixel attack for fooling deep neural networks.
       "IEEE Transactions on Evolutionary Computation", 2019. [Online].
       Available: `arXiv:1710.08864 [stat.ML] <https://arxiv.org/abs/1710.08864>`_.

```
Restore removed whitespace to ensure restructured text parser doesn't get confused.